src/scss/_badges.scss

```diff
@@ -20,6 +20,21 @@
 	color: $text-primary;
 }
 
+.label-default {
+	background-color: $grey-6;
```

Comment:
@fuhton I submitted my own PR by accident. You can disregard mine but for this area, let's go with my version. If you just overwrite the existing Bootstrap variable - it will automatically set this default color without having to write extra styles.

https://github.com/JDRF/design-system/pull/252